cluster_tools/copy_volume/copy_volume.py

```diff
@@ -116,15 +116,16 @@ def run_impl(self):
 
         compression = task_config.pop("compression", "gzip")
 
+        dtype = DTYPE_MAPPING.get(dtype, dtype)
```

Comment:
p.s.: I can only emphasize again to use a linter, this will for example catch issues like this without running any tests because you are using a variable without being previously declared here.